test/apiv2/12-imagesMore.at

```diff
@@ -25,12 +25,7 @@ t GET libpod/images/$IMAGE/json 200 \
 # Run registry container
 podman run -d --name registry -p 5000:5000 quay.io/libpod/registry:2.6 /entrypoint.sh /etc/docker/registry/config.yml
 
-# Push to local registry
-# FIXME: this is failing:
-#   "cause": "received unexpected HTTP status: 500 Internal Server Error",
-#   "message": "error pushing image \"localhost:5000/myrepo:mytag\": error copying image to the remote destination: Error writing blob: Error initiating layer upload to /v2/myrepo/blobs/uploads/ in localhost:5000: received unexpected HTTP status: 500 Internal Server Error",
-#   "response": 400
-#t POST libpod/images/localhost:5000/myrepo:mytag/push\?tlsVerify\=false '' 200
+t POST "images/localhost:5000/myrepo/push?tlsVerify=false&tag=mytag" '' 200
```

Comment:
I'm somewhat sure the test failure is a race condition: the podman registry is still in the process of coming up.

Could you add a `sleep 2` just before this, and see if it helps? If it does, I will follow up with a cleaner `wait-for-port` helper function.

And while you're at it, please revert line 28, I think that's a helpful comment.

Reply:
ah my machine is too fast!

Reply:
<b>works on my machine<sup>TM</sup></b>

Reply:
Mine too. But our CI systems... a.r.e.. v.e.r.y.. s.l.o.w...  hence my suggestion. Thanks for understanding.

Reply:
@edsantiago maybe it's not about fast machine, maybe I had registry running from previous run on my machine.

Reply:
Well, the test passed on my laptop, and I didn't/don't have a registry (or anything) on :5000

Reply:
Have you at least had `quay.io/libpod/registry:2.6` in local registry? It takes a while to download.

Reply:
Oh while the image is being pulled it should be blocked, ignore the comment above.